Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -367,6 +369,8 @@ public async Task<User> AuthenticateUser(string username, string password, strin
             return success ? user : null;
         }
 
+#nullable enable
```

Comment:
what this means?

Reply:
This does the same as #1942 but only for a specific region. In the end we would want all code to be covered by this. I'm just not sure how we should undertake that. We can either do it like #1942 whole projects at a time (by excluding all files and adding them in 1by one). Or the other way around, adding regions until the whole solution is covered.